src/View/mainpage/IsodoseTab.py

```diff
@@ -1,189 +1,160 @@
-		self.checkbox6.setStyleSheet("font: 10pt \"Laksaman\";")
-		self.checkbox7.setStyleSheet("font: 10pt \"Laksaman\";")
-		self.checkbox8.setStyleSheet("font: 10pt \"Laksaman\";")
-		self.checkbox9.setStyleSheet("font: 10pt \"Laksaman\";")
-		self.checkbox10.setStyleSheet("font: 10pt \"Laksaman\";")
-
-	# Function triggered when a dose level selected
-	# Updates the list of selected isodoses and dicom view
-	def checked_dose(self, state, isod_value):
-		"""
-		Function triggered when the checkbox of a structure is checked / unchecked.
-		Update the list of selected structures.
-		Update the DICOM view.
-		
-		:param state: True if the checkbox is checked, False otherwise.
-		:param isod_value: Percentage of isodose.
-		"""
-		if state:
-			# Add the dose to the list of selected doses
-			self.main_window.selected_doses.append(isod_value)
-		else:
-			# Remove dose from list of previously selected doses
-			self.main_window.selected_doses.remove(isod_value)
-		# Update the dicom view
-		self.main_window.dicom_view.update_view()
-
-	def init_color_square(self):
-		"""
-		Initialize the color squares.
-		"""
-		self.color1_isod = self.draw_color_square(131, 0, 0)
-		self.color2_isod = self.draw_color_square(185, 0, 0)
-		self.color3_isod = self.draw_color_square(255, 46, 0)
-		self.color4_isod = self.draw_color_square(255, 161, 0)
-		self.color5_isod = self.draw_color_square(253, 255, 0)
-		self.color6_isod = self.draw_color_square(0, 255, 0)
-		self.color7_isod = self.draw_color_square(0, 143, 0)
-		self.color8_isod = self.draw_color_square(0, 255, 255)
-		self.color9_isod = self.draw_color_square(33, 0, 255)
-		self.color10_isod = self.draw_color_square(11, 0, 134)
-
-
-	def init_color_isod(self):
-		"""
-		Create a dictionary containing the colors for each isodose.
-
-		:return: Dictionary where the key is the percentage of isodose and the value a QColor object.
-		"""
-		roiColor = dict()
-		roiColor[107] = QtGui.QColor(131, 0, 0)
-		roiColor[105] = QtGui.QColor(185, 0, 0)
-		roiColor[100] = QtGui.QColor(255, 46, 0)
-		roiColor[95] = QtGui.QColor(255, 161, 0)
-		roiColor[90] = QtGui.QColor(253, 255, 0)
-		roiColor[80] = QtGui.QColor(0, 255, 0)
-		roiColor[70] = QtGui.QColor(0, 143, 0)
-		roiColor[60] = QtGui.QColor(0, 255, 255)
-		roiColor[30] = QtGui.QColor(33, 0, 255)
-		roiColor[10] = QtGui.QColor(11, 0, 134)
-		return roiColor
-
-
-	def draw_color_square(self, red, green, blue):
-		"""
-		Create a color square.
-		:param red: Red component of the color.
-		:param green: Green component of the color.
-		:param blue: Blue component of the color.
-
-		:return: Color square widget.
-		"""
-		color_square_label = QtWidgets.QLabel()
-		color_square_pix = QtGui.QPixmap(15, 15)
-		color_square_pix.fill(QtGui.QColor(red, green, blue))
-		color_square_label.setPixmap(color_square_pix)
-		return color_square_label
\ No newline at end of file
+from PyQt5 import QtWidgets, QtGui, QtCore
+
+from src.Model.PatientDictContainer import PatientDictContainer
+
```

Comment:
the docstring got stripped out.
They should be put back in and updated if need be.